api/libproject/generate_appengine_cron_yaml.rb

```diff
@@ -0,0 +1,20 @@
+# Generates the App Engine cron.yaml file for a given environment. This is meant
```

Comment:
Sorry for the late review, but this is easier than trying to cross-ref stuff from Slack.

First question I had was, why does this need to be a Ruby script? If it's just doing file operations, there's support for that natively in gradle. Also, it looks like Gradle supports Kotlin now in addition to Groovy, so maybe there's an opportunity for tighter integration. It doesn't look like this script will be doing much outside of gradle tasks.

I realize there are future plans here, and if Ruby has a better Yaml parser than you get out of the boxk, then that makes sense.

Also, anecdotally, we're seeing problems with gradle tasks that call ruby. With this commit I can't run `gradlew appengineRun` anymore on my Mac.

Reply:
The rationale for using Ruby here is for consistency. There's a highly similar script in this same directory called for a similar purpose by Gradle. Moving both of them to Groovy might be fine, I haven't evaluated it so I don't know whether that would be worth the effort. I think in the future we'll want to do more munging of the yaml in this script, so we should ensure that this process is straightforward in whichever scripting language. 

My fault, sorry for the breakage. Per slack, please run `source api/db/load_vars.sh` to fix.